apps/vast/pyvast_threatbus/pyvast_threatbus.py

```diff
@@ -494,12 +494,10 @@ async def live_match_vast(
     @param vast_binary The VAST binary command to use with PyVAST
     @param vast_endpoint The endpoint of a running VAST node
     @param sightings_queue The queue to put new sightings into
-    @param retro_match Boolean flag to use retro-matching over live-matching
     """
     global logger, matcher_name
     vast = VAST(binary=vast_binary, endpoint=vast_endpoint, logger=logger)
-    matcher_name = "threatbus-" + "".join(random.choice(letters) for i in range(10))
-    proc = await vast.matcher().start(name=matcher_name).exec()
```

Comment:
Where do you start the matcher if not here?

Reply:
The idea is that the matcher should already be running in VAST, and we only attach here. This would be done e.g. in the VAST config.